include/seqan3/io/structure_file/input.hpp

```diff
@@ -1098,10 +1098,10 @@ namespace std
 /*!\brief std::tuple_size overload for column-like access.
  * [metafunction specialisation for seqan3::structure_file_in]
  */
-template<seqan3::structure_file_input_traits_concept traits_type,
-         seqan3::detail::fields_concept selected_field_ids,
+template<seqan3::StructureFileInputTraits                    traits_type,
+         seqan3::detail::Fields                              selected_field_ids,
          seqan3::detail::TypeListOfStructureFileInputFormats valid_formats,
-         seqan3::char_concept stream_char_t>
+         seqan3::char_concept                                stream_char_t>
```

Comment:
Can you change this one to `std::Integral` as the char_concept is being removed anyway?